docs/docs/guide/2_api-reference/cms/registerCmsElement.md

```diff
@@ -0,0 +1,25 @@
+# Register CMS element
```

Comment:
Can you also add a screenshot of an example result. This makes it easier for the user to understand what it will look like